app/javascript/dashboard/routes/dashboard/commands/inboxHotKeys.js

```diff
@@ -61,14 +64,28 @@ const INBOX_SNOOZE_EVENTS = [
   },
 ];
 export default {
+  data() {
+    return {
+      showSnoozeNotificationItems: false,
+    };
+  },
   computed: {
     inboxHotKeys() {
-      if (isAInboxViewRoute(this.$route.name)) {
+      if (
+        isAInboxViewRoute(this.$route.name) ||
+        this.showSnoozeNotificationItems
+      ) {
         return this.prepareActions(INBOX_SNOOZE_EVENTS);
       }
       return [];
     },
   },
+  mounted() {
+    bus.$on(CMD_TOGGLE_SNOOZE_NOTIFICATION, this.toggleSnoozeOptions);
+  },
+  destroyed() {
+    bus.$off(CMD_TOGGLE_SNOOZE_NOTIFICATION, this.toggleSnoozeOptions);
```

Comment:
Shouldn't this value of showSnoozeNotificationItems be false on the destroyed event?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/8982/commits/685e5c615b9657a1d8878faa02c3efc4f46e184c